### ISSUE TITLE
GitHub Actions caching does not appear to work for Windows

### ISSUE THREAD

**Issue by lhstrh** [lf-lang/lingua-franca#1664]
But I'm unsure why...

**Comment by lhstrh** [comment]
It appears that the problem is this:
```
"C:\Program Files\Git\usr\bin\tar.exe" --posix -cf cache.tzst --exclude cache.tzst -P -C D:/a/lingua-franca/lingua-franca --files-from manifest.txt --force-local --use-compress-program "zstd -T0"
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/7.6.1/executionHistory/executionHistory.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/7.6.1/fileContent/fileContent.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/7.6.1/fileHashes/fileHashes.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/7.6.1/generated-gradle-jars/generated-gradle-jars.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/7.6.1/javaCompile/javaCompile.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/jars-9/jars-9.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/journal-1/journal-1.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: C\:/Users/runneradmin/.gradle/caches/modules-2/modules-2.lock: Read error at byte 0, while reading 38 bytes: Device or resource busy
/usr/bin/tar: Exiting with failure status due to previous errors
Warning: Failed to save: "C:\Program failed with error: The process 'C:\Program Files\Git\usr\bin\tar.exe' failed with exit code 2
```

**Comment by lhstrh** [comment]
This has been fixed in #1670.
